t/features/wildcard.feature

```diff
@@ -0,0 +1,78 @@
+
+Feature: analizing projects with WildCard
+    As a software developer
+    I want analizo to correctly analize projects with WildCard
+    So that I can evaluate it
+
+    Scenario: "WildCard" project metrics number of modules
+        Given I am in t/samples/wildcard/<language>
+        When I run "analizo metrics ."
+        Then analizo must report that the project has total_modules = <total_modules>
+        Examples:
+            | language | total_modules |
+            | java     | 2             |
+
+    Scenario: "WildCard" project metrics modules name
+        Given I am in t/samples/wildcard/<language>
+        When I run "analizo metrics ."
+        Then analizo must report that file <filename> declares module <wildcard_class>
+        Examples:
+            | language | filename              | wildcard_class |
+            | java     | Wildcard_test.java    | Wildcard_sample |
+            | java     | Wildcard_test.java    | GenericClass |
+
+    Scenario: "WildCard" project metrics
+        Given I am in t/samples/wildcard/<language>
```

Comment:
why creates samples on t/samples/wildcard/java if the only sample code is for Java lang? I changed the PR in order to avoid this, I moved t/samples/wildcard/java/* to t/samples/wildcard/.

Reply:
Wildcard is present ir other languages including cpp. Our goal was to leave this structure open to further updates to other languages.